community_calendar/api.py

```diff
@@ -7,10 +7,21 @@
 import datetime
 from django.utils import timezone
 
+from django.contrib.auth.models import User
 from .models import Event
 
+
 # Serializers define the API representation.
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['first_name', 'last_name']
```

Comment:
@s-kennedy I'm hesitant to include last names in public APIs. I've removed last_name.